README.md

```diff
@@ -71,6 +73,21 @@ vars:
     product_pass_through_columns: []
     product_variant_pass_through_columns: []
 ```
+
+### Disable Models
+This package was designed with the intention that users have all relevant Shopify tables being synced by Fivetran. However, if you are Shopify user that does not operate on returns or adjustments then you will not have the related source tables. As such, you may use the below variable configurations to disable the respective downstream models. All variables are `true` by default, only add the below configuration if you are wishing to disable the models:
+
+```yml
+# dbt_project.yml
+
+...
+vars:
+  shopify_source:
```

Comment:
should we not include the source package scope in this snippet, since these variables are referenced in the transform package?

Reply:
Updated to be global. Good call out!